src/Api.php

```
@@ -146,6 +156,8 @@ public function api_listener(): void {
 				$this->get_woocommerce_status_list();
 			} elseif ( isset( $action ) && 'woocommerce_order_status_update' === $action ) {
 				$this->woocommerce_order_status_update( $this->order_id, $this->order_status );
+			} elseif ( isset( $action ) && 'woocommerce_subscription_cancel' === $action ) {
```

PHPStan (2.2.2)

[error] Variable $action in isset() always exists and is not nullable.


(isset.variable)